ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/impl/ConnectorCommandResponseHandler.java

```diff
@@ -33,7 +33,8 @@ static void handleCreateConnectorResponse(
       final JsonObject connectorInfoEntity,
       final CompletableFuture<Void> cf
   ) {
-    if (connectorInfoEntity.getString("@type").equals("connector_info")) {
+    if (connectorInfoEntity.containsKey("statementText")
+        && connectorInfoEntity.containsKey("info")) {
```

Comment:
Changed the logic for this because it was failing in the unit tests.

Reply:
To check my understanding: you're saying that this works in integration tests because of how the response entity is serialized and deserialized, but doesn't work in unit tests since those don't go through the serialization/deserialization path? 

I'm fine with switching away from checking for `@type`. However, let's reuse `AdminResponseHandlers#isCreateConnectorResponse()` and similar for the others, rather than duplicating the logic in two places.

Reply:
Exactly